Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -41,6 +41,11 @@ struct ContextParameter {
     } value;
 };
 
+enum class MaterialFace : size_t {
```

Comment:
```suggestion
enum class MaterialFace : u8 {
```

?

Reply:
This ends up implicitly converted to `size_t` anyway when passed into `AK::Array::at()`, so I thought I may as well keep the typing the same, but I guess `u8` probably makes more sense :thinking: